ui/public/index.html

```diff
@@ -12,7 +12,7 @@
   <meta name="theme-color" content="#ffffff"/>
   <meta
       name="description"
-      content="Machine Learning Platform"
+      content="CaraML"
```

Comment:
Updated this to make the migration consistent with the future changes to the MLP UI i.e. renaming of 'Machine Learning Platform' to 'CaraML'.